packages/core/src/Stores/client-store.js

```diff
@@ -295,6 +295,46 @@ export default class ClientStore extends BaseStore {
         );
     }
 
+    @computed
+    get is_europe() {
+        const eu_shortcode_regex = new RegExp('^(maltainvest|malta|iom)$');
+        const eu_excluded_regex = new RegExp('^mt$');
```

Comment:
Please move these RegExp declarations somewhere where they can be static. I believe it's very expensive to declare an expression on every computation of `@computed`.

Reply:
@mahdipourziaei-binary But if I remove them, I need to add a more complex if/else condition. what is your suggestion as an alternative?

Reply:
@mahdiaryayi-fs I did not mean to remove, just to move the `const my_regexp` declaration out of this `computed` function. So, for example, move it to the top of the file, where the RegExp declaration is made only once.